.github/workflows/playwright.yml

```
@@ -95,6 +95,8 @@ jobs:
       - name: Start frontend
         working-directory: ./frontend
         run: yarn start &> frontend.log &
+        env:
+          STRIPE_PUBLIC_KEY: ${{ env.STRIPE_DEV_PUBLISHABLE_KEY }}
```

This should become:
```suggestion
          STRIPE_PUBLISHABLE_KEY: ${{ env.STRIPE_DEV_PUBLISHABLE_KEY }}
```
once the code is updated